src/views/DashboardView/MyTasks.js

```diff
@@ -73,7 +73,7 @@ const MyTasks = ({ className, ...rest }) => {
             backgroundColor: colors.common.white
           }}
           cardStyle={{
-            backgroundColor: colors.deepOrange[200],
+            backgroundColor: '#ebecf0',
```

Comment:
Can we change the color gradients hex-code here, if we not using ditto deepOrange?

I have added a `customColors` file, just replace the color gradients with the new one, ie `#ebecf0`. Then it can be easily imported and used as `colors.MyNewColor[200]` or `colors.MyNewColor[400]`.

This will help us keep the color codes consistent across the project